test/blackbox-tests/test-cases/variants/run.t

```diff
@@ -1,16 +1,17 @@
 Variant feature is auto enabled when virtual_modules is used
 
   $ dune build --root variants-without-using
-  File "dune", line 3, characters 1-25:
-   (virtual_modules foobar))
-   ^^^^^^^^^^^^^^^^^^^^^^^^
-  Error: 'virtual_modules' is only available since version 0.1 of the experimental variants feature
-  [1]
```

Comment:
Why is this error gone?

Reply:
So actually, I think that I simply uncovered an unrelated bug in this PR. I trivially forgot to set the version number correctly (it was `(0, 0)` rather than `(0, 1)` and that uncovered this bug. I think the gist of it is that the latest version of variants is being automatically selected. This behavior shouldn't be available for experimental plugins. I will fix this in another PR.